jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttonGroup/ToolbarTests.java

```diff
@@ -63,10 +63,10 @@ public void buttonsInButtonToolbarTest() {
 
     @Test
     public void inputFieldInButtonToolbarTest() {
-        buttonToolbarWithInputField.inputAreaInToolbar.is().displayed();
-        buttonToolbarWithInputField.inputAreaInToolbar.is().enabled();
-        buttonToolbarWithInputField.inputAreaInToolbar.is().core().attr("placeholder", placeholderForInputField);
-        buttonToolbarWithInputField.inputAreaInToolbar.setValue(textForTestingInputField);
-        assertEquals(buttonToolbarWithInputField.inputAreaInToolbar.getValue(), textForTestingInputField);
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().displayed();
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().enabled();
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.is().core().attr("placeholder", placeholderForInputField);
+        buttonGroupToolbarWithInputField.inputAreaInToolbar.setValue(textForTestingInputField);
+        assertEquals(buttonGroupToolbarWithInputField.inputAreaInToolbar.getValue(), textForTestingInputField);
     }
 }
```

Comment:
Please add new line at the end of the file

Reply:
done.